app/views/content/non-uk-teachers/visas-for-non-UK-trainees.md

```diff
@@ -0,0 +1,257 @@
+
+If your visa application is successful, you will be able to train as a teacher on either a salaried or fee-paying initial teacher training course, as long as the course does not meet the requirements for a student visa. If the course can sponsor a student visa, you must apply for this type of visa.
+
+### The Youth Mobility Scheme visa 
+
+The [Youth Mobility Scheme](https://www.gov.uk/youth-mobility) allows [young people from certain countries](https://www.gov.uk/youth-mobility/eligibility) to come to the UK to study or work without a sponsor. This includes training as a teacher through either a salaried or fee-paying initial teacher training course.
+
+A Youth Mobility Scheme visa lasts up to 2 years. You may be able to switch from a Youth Mobility Scheme visa to another visa route (for example, the skilled worker visa) without having to leave the UK. You’ll need to meet the requirements for the new visa route.
+
+### Other visas 
+Depending on your personal circumstances, you may be able to work or study in the UK on alternative visas – for example:
+
+* a [Family visa](https://www.gov.uk/uk-family-visa)
+* a [UK Ancestry visa](https://www.gov.uk/ancestry-visa) 
+* if you are from Hong Kong, a [British National (Overseas) visa](https://www.gov.uk/british-national-overseas-bno-visa)
+
+### Refugees and asylum seekers 
+
+If you have been granted refugee status, you can apply for teacher training – either salaried or fee-paying – and will usually be eligible for [financial support to train to teach on a fee-paying course](/non-uk-teachers/fees-and-funding-for-non-UK-trainees).
+
+If you are in the process of seeking asylum, you should refer to your immigration bail conditions to see if you have permission to study on a teacher training course. You will not usually be eligible for financial support to train to teach on a fee-paying course. You are also unlikely to be able to undertake salaried teacher training while your asylum claim is considered, because asylum seekers do not usually have the right to work, although [there are some exceptions](https://www.gov.uk/government/publications/working-whilst-an-asylum-claim-is-considered/working-in-the-uk-whilst-an-asylum-case-is-considered).
+
+Learn more about:
+
+* [claiming asylum in the UK](https://www.gov.uk/claim-asylum) if you are an asylum seeker
+* [support for British and non-British nationals in Afghanistan](https://www.gov.uk/guidance/support-for-british-and-non-british-nationals-in-afghanistan)
+* [support for Ukrainian nationals](/non-uk-teachers/ukraine)
+
+## Visa fees and other charges
+
+You will usually need to show proof of a certain level of savings or income, depending on which visa you are applying for. You’ll be guided through this process when you apply for your visa.
+
+### Immigration Health Surcharge (IHS)
```

Comment:
Should visa fees come first as per the h2?

Reply:
yes good point